src/registry.json

```diff
@@ -2597,12 +2597,14 @@
         "category": "interoperability",
         "support": {
           "contact": "https://help.fio.net",
-          "knowledgeBase": "https://app.fio.net/metamask",
           "faq": "https://app.fio.net/metamask"
         },
         "sourceCode": "https://github.com/fioprotocol/fio-wallet-snap"
       },
       "versions": {
+        "1.1.1": {
```

Comment:
Should we hold off merging this until name resolution is rolled out?

Reply:
The feature is already live. When the category rolls out I will want to change the category on this one, but for now it can be allowlisted in its current category.

Reply:
I thought it was in 12.4? https://github.com/MetaMask/metamask-extension/pull/26393

Reply:
Yeah now I know